css/styles.css

```diff
@@ -47,62 +44,276 @@ nav {
   padding: 0;
 }
 
-.container-1 {
-  width: 100%;
-  height: 30px;
+hr {
+  width: 85%;
+  margin: 10px auto 10px auto;
+  padding: 0;
+  border-bottom: solid 2px #dfe1e6;
 }
 
-.container-2 {
+.poppins {
   font-family: Poppins, sans-serif;
-  margin-left: 0;
-  margin-right: 48px;
+}
+
+/* Toolbar */
+.container-2 {
+  margin-left: 27px;
+  margin-right: 37px;
+  margin-top: 10px;
   display: flex;
-  align-items: center;
   justify-content: space-between;
+  align-items: center;
 }
 
+/* Section Headline */
 .headline {
   margin: 20px, 0, 20px, 0;
-  font-family: Poppins, sans-serif;
   color: #172b4d;
-  width: 375px;
+  width: 100%;
   display: flex;
   flex-direction: column;
 }
 
 .logo-txt {
-  font-family: Poppins, sans-serif;
   color: #6070ff;
   font-weight: bold;
   font-size: 18px;
 }
 
 .name {
   width: 85%;
-  margin-top: 20px;
+  margin: 20px 20px 0 24px;
+  color: #172b4d;
 }
 
 .desc {
-  margin-top: 14px;
+  margin: 16px 10px 20px 24px;
   font-weight: 400;
   width: 85%;
-  line-height: 1.5;
-  text-align: justify;
+  color: #344563;
+  line-height: 1.5em;
+}
+
+.connect {
+  width: 85%;
+  margin: 0 20px 15px 24px;
+  color: #7f8cff;
+  font-weight: 500;
 }
 
 .container-3 {
   display: flex;
+  margin: 0 20px 20px 24px;
 }
 
-.connect {
-  margin-top: 20px;
+/* Section Works */
+.works {
+  width: 100%;
+}
+
+.grid-1 {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 640px 700px 640px 700px;
+  gap: 25px;
+}
+
+.work1 {
+  background-color: #fff;
+  border: solid 1px #dfe1e6;
+  border-radius: 16px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.desc2 {
+  font-size: 16px;
+  font-weight: 500;
+  color: #344563;
+  line-height: 1.5em;
+  margin-left: 5%;
+  margin-right: auto;
+  margin-bottom: 15px;
+}
+
+.work1-img {
+  display: block;
+  width: 90%;
+  height: auto;
+  margin-top: 15px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.work-text1 {
+  margin-top: 8px;
+  font-size: 32px;
+  color: #172b4d;
+  width: 70%;
+  margin-left: 5%;
   margin-bottom: 10px;
-  color: #7f8cff;
 }
 
-.linkedin.m.github.sign {
+.canopy {
+  color: #344563;
+  font-size: 12px;
+  font-weight: 600;
+  margin: 0;
+  padding: 0;
+}
+
+.bed,
+.year {
+  color: #7a869a;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.dot {
+  margin: 0;
+  padding: 0;
+  width: 60%;
+  height: auto;
+}
+
+.stack {
+  font-size: 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  padding: 4px 12px 4px 12px;
+  border-radius: 4px;
+}
+
+.flex1,
+.flex2 {
+  margin-left: 5%;
+  margin-right: auto;
+  display: flex;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.see-project {
+  font-size: 17px;
+  color: #396df2;
+  border: solid 1px #396df2;
+  padding: 8px;
+  border-radius: 4px;
+  margin-left: 5%;
+  margin-right: auto;
+}
```

Comment:
Your project looks great. Kindly adjust the background color and padding within the buttons to match that of the design. You seemed to missed out assigning one.

Your design:
![image](https://user-images.githubusercontent.com/20554583/228999401-6655dfff-402f-40c5-ac10-93ab00e354f1.png)    ![image](https://user-images.githubusercontent.com/20554583/228999711-07a02f28-d80b-46db-a07d-9d7e80e3641d.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/228999450-7acfe048-e9b7-4608-8e87-89523e0ec052.png)   ![image](https://user-images.githubusercontent.com/20554583/228999745-fa9e63cb-182f-4d76-bc3e-7b782d99217e.png)